buildingmotif/dataclasses/library.py

```diff
@@ -112,7 +112,7 @@ def create(cls, name: str, overwrite: Optional[bool] = True) -> "Library":
             if overwrite:
                 cls._clear_library(db_library)
             else:
-                logging.warn(
+                logging.warning(
```

Comment:
should these be `warnings.warn`? I know I've talked about this before how we should decide on which one we are using throughout the codebase.

Reply:
`logging.warn` is deprecated now, and using `logging.warning` is the recommended fix. I can't really find a ton of articles on using `warning` vs `logging` package to do the warnings. I figured because we were already using `logging` for other stuff, we should just continue to use that package here

Reply:
After doing some research I think `warning` is more meant for user facing warnings.

what I meant with this comment is that we are using both `warning.warn` and `logging.warning` in different parts of the code. where they seem a little redundant? they're both native python libraries so it's not a big deal either way, I just want to make sure we're consistent with our warnings.